### PR TITLE
[Fixes] Further format alignment and fix `@maxLength(80)` for some resource groups

### PR DESCRIPTION
**Pull request by eriqua**
# Description

>Thank you for your contribution !

> Please include a summary of the change and which issue is fixed.
> Please also include the context.
> List any dependencies that are required for this change.

| Pipeline |
| - |
| [AnalysisServices: Servers](https://github.com/Azure/ResourceModules/actions/workflows/ms.analysisservices.servers.yml) |

# Type of Change

Please delete options that are not relevant.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

# Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep [hunk 1]
```diff
@@ -51,3 +52,4 @@ module testDeployment '../../deploy.bicep' = {
     ]
   }
 }
+}
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:55]: Intentional?

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep [hunk 2]
```diff
@@ -46,3 +47,4 @@ module testDeployment '../../deploy.bicep' = {
     sqlAdministratorLogin: 'synwsadmin'
   }
 }
+}
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:50]: Intentional?

Comment (eriqua) [reply modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep]: fixed

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep (hunk 1 above)
Comment (eriqua) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep]: fixed

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**